pkg/option/flags.go

```diff
@@ -416,4 +419,6 @@ func AddFlags(flags *pflag.FlagSet) {
 	flags.Int(KeyEventCacheRetryDelay, defaults.DefaultEventCacheRetryDelay, "Delay in seconds between event cache retries")
 
 	flags.Bool(KeyCompatibilitySyscall64SizeType, false, "syscall64 type will produce output of type size (compatibility flag, will be removed in v1.4)")
+
+	flags.String(KeyExecveMapEntries, "", "Set entries for execve_map table (default 32768)")
```

Comment:
Can we have the default from the actual default value here? If we ever change it, the change should be reflected in help.